pkg/cspc/algorithm/select_node.go

```diff
@@ -43,7 +43,7 @@ func (ac *Config) SelectNode() (*cstor.PoolSpec, string, error) {
 		pool := pool
 		nodeName, err := ac.GetNodeFromLabelSelector(pool.NodeSelector)
 		if err != nil || nodeName == "" {
-			klog.Errorf("could not use node for selectors {%v}", pool.NodeSelector)
+			klog.Errorf("could not use node for selectors {%v}: {%s}", pool.NodeSelector, err)
```

Comment:
Let us use format specifier for error as %v (or) change argument from err.Error()